src/primitives/block.cpp

```diff
@@ -107,6 +120,11 @@ uint256 CBlock::BuildScMerkleRootsMap()
         tx.getCrosschainOutputs(mScMerkleTreeLeaves);
     }
 
+    BOOST_FOREACH(const CScCertificate& cert, vcert)
```

Comment:
Modify hash calculation accordingly to Zendoo paper

Reply:
Done